packages/opentelemetry-instrumentation/src/platform/node/instrumentation.ts

```diff
@@ -59,12 +59,12 @@ export abstract class InstrumentationBase<T = any>
     }
   }
 
-  private _onRequire<T>(
-    module: InstrumentationModuleDefinition<T>,
-    exports: T,
+  private _onRequire<U>(
```

Comment:
I don't mind but I'm curious why you changed T to U

Reply:
The issue was that type parameter of the method was shadowing the type parameter of the class

Reply:
Nit: it looks weird tbh, I would use some longer name maybe `InstrumentationModuleType` etc.